src/services/refact.ts

```diff
@@ -200,3 +200,37 @@ export type CapsResponse = {
   tokenizer_rewrite_path: Record<string, unknown>;
   chat_rag_functions?: string[];
 };
+export type RefactTableImpactDatesRow = {
+  [key in ColumnName]: number;
+};
+export type RefactTableImpactLanguagesRow = {
+  [key in ColumnName]: string | number;
+};
+export type RefactTableData = {
```

Comment:
I guess type-guards can be added later when some IO happens.... which reminds me I'll need to look in to runtime validation libraries

Reply:
I need to use types in order not to have type errors

Reply:
👍🏻  eeyup, i was talking about run time type checking, theses types don't need right now, but when calls get made to the back end it might :)